pkg/cmd/pulumi/stack.go

```diff
@@ -140,7 +142,8 @@ func newStackCmd() *cobra.Command {
 				outputs, err := getStackOutputs(snap, showSecrets)
 				if err == nil {
 					fmt.Printf("\n")
-					printStackOutputs(outputs)
+					_ = fprintStackOutputs(os.Stdout, outputs)
+					// stdout error ignored
```

Comment:
```contract.AssertNoError(err)``` ? If this is we really really don't expect stdout to ever actually error on us

Reply:
I'm not sure about using contract.AssertNoError here.
I don't think it'll add any value.
We ignore write-to-stdout errors everywhere else (e.g. the line above this).
If we do run into an error writing to stdout, a panic on this specific statement won't make it better.

This `_ = fn()` thing is just to appease errcheck (it has printf in its [list of exclusions](https://github.com/kisielk/errcheck/blob/master/errcheck/excludes.go#L14)) — this felt better than a `//nolint` comment.

That said, if you feel strongly that we should panic on a non-nil error here, I'm happy to change this.
It made sense to provide good reasons to ignore the error first.

Reply:
eh it's not a strong opinion, if we're ignoring these everywhere else anyway don't sweat it here.